README.md

```diff
@@ -128,7 +128,7 @@ project, making adoption of _pydoclint_ much easier.
 Please read this page:
 [How to configure _pydoclint_](https://jsh9.github.io/pydoclint/how_to_config.html)
 
-### 2.6. How to ignore certain violations in _flake8_ mode
+### 2.6. How to ignore certain violations
```

Comment:
We can ignore violations only in flake8 mode, so I think the word "in _flake8_ mode" is necessary here.

Reply:
I think the "how to ignore" page as of this PR describes both routes, was why I got rid of the "in flake8 mode" part. However, I can also see what you're saying, so I reverted this change